src/Types.mo

```diff
@@ -89,6 +89,6 @@ module {
     public type Map<K, V> = (); // Placeholder
     public type Queue<T> = (Stack.Stack<T>, Stack.Stack<T>);
```

Comment:
If the intention is to keep the old functional Queue from base, then this should be:
```suggestion
    public type Queue<T> = (Stack<T>, Stack<T>);
```
(otherwise I think it's using the imperative stack.

Reply:
Oh that's a good point; this was a copy/paste error. Thanks for catching this.